integ/persistence_test.go

```diff
@@ -148,13 +155,31 @@ func doTestPersistenceWorkflow(
 			SearchAttributes: []iwfidl.SearchAttribute{
 				expectedDatetimeSearchAttribute,
 			},
+			DataAttributes: []iwfidl.KeyValue{
+				expectedDataAttribute,
+			},
 			WorkflowConfigOverride:   config,
 			UseMemoForDataAttributes: ptr.Any(useMemo),
 		},
 	}
 	_, httpResp, err := reqStart.WorkflowStartRequest(wfReq).Execute()
 	panicAtHttpError(err, httpResp)
 
+	// TODO: Fix the issue with running queryHandler before new workflow which Continues as New has started
+	time.Sleep(time.Millisecond * 10)
```

Comment:
Looks like the test failed. I looked at my other sleeps for the same reasons, I used 1 second for safety. Also it's better to only sleep for continueAsNew cases. (the test will run faster that way)